scripts/benchmark/checks/deploy_dos.py

```diff
@@ -0,0 +1,314 @@
+
+        assert computation.is_success
+
+        # Interact with the deployed contract by calling the totalSupply() API ?????
+        self.dos_contract = self.w3.eth.contract(
+            address=Web3.toChecksumAddress(encode_hex(self.deployed_contract_address)),
+            abi=self.contract_interface['abi'],
+        )
+    def sstore_uint64(self, chain: MiningChain) -> None:
+        w3_tx2 = self.dos_contract.functions.storageEntropy().buildTransaction(W3_TX_DEFAULTS) 
+
+        tx2 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=FUNDED_ADDRESS_PRIVATE_KEY,
+            from_=FUNDED_ADDRESS,
+            to=self.deployed_contract_address,
+            amount=0,
+            gas=SECOND_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx2['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx2)
+
+        assert computation.is_success
+
+    def create_empty_contract(self, chain: MiningChain) -> None:
+        w3_tx3 = self.dos_contract.functions.createEmptyContract().buildTransaction(W3_TX_DEFAULTS) 
+
+        tx3 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=FUNDED_ADDRESS_PRIVATE_KEY,
+            from_=FUNDED_ADDRESS,
+            to=self.deployed_contract_address,
+            amount=0,
+            gas=THIRD_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx3['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx3)
+
+
+        assert computation.is_success
+
+    def sstore_uint64_revert(self, chain: MiningChain) -> None:
+        w3_tx4 = self.dos_contract.functions.storageEntropyRevert().buildTransaction(W3_TX_DEFAULTS) 
+
+        tx4 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=FUNDED_ADDRESS_PRIVATE_KEY,
+            from_=FUNDED_ADDRESS,
+            to=self.deployed_contract_address,
+            amount=0,
+            gas=FORTH_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx4['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx4)
+
+        # assert computation.is_success
+    def create_empty_contract_revert(self, chain: MiningChain) -> None:
+        w3_tx5 = self.dos_contract.functions.createEmptyContractRevert().buildTransaction(W3_TX_DEFAULTS) 
+
+        tx5 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=FUNDED_ADDRESS_PRIVATE_KEY,
+            from_=FUNDED_ADDRESS,
+            to=self.deployed_contract_address,
+            amount=0,
+            gas=FIFTH_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx5['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx5)
+
+class DosContractDepolyBenchmark(BaseDoScontractBenchmark):
+    def __init__(self) -> None:
+        super().__init__()
+
+    @property
+    def name(self) -> str:
+        return 'DoScontract deployment'
+
+    def _setup_benchmark(self, chain: MiningChain) -> None:
```

Comment:
let's make `_setup_benchmark` non-abstract in the base contract and then we don't need this empty implementation here. In other words, provide the empty implementation in the base contract.